### DISCUSSION TITLE
AttributeError: Can't get attribute 'DataEdgeAttr'

### DISCUSSION THREAD

**Discussion by LukeLIN-web** [pyg-team/pytorch_geometric#5862] [Q&A] [answered]
I get a docker images. 
pip list show it has 
torch                  1.9.0
torch-geometric        2.0.2
but  when I try
```python
from torch_geometric.datasets import Reddit

datapath = "/root/share/data/Reddit"
dataset = Reddit(datapath)
```

  File "/opt/conda/lib/python3.7/site-packages/torch_geometric/datasets/reddit.py", line 33, in __init__
    self.data, self.slices = torch.load(self.processed_paths[0])
  File "/opt/conda/lib/python3.7/site-packages/torch/serialization.py", line 607, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/opt/conda/lib/python3.7/site-packages/torch/serialization.py", line 882, in _load
    result = unpickler.load()
  File "/opt/conda/lib/python3.7/site-packages/torch/serialization.py", line 875, in find_class
    return super().find_class(mod_name, name)
AttributeError: Can't get attribute 'DataEdgeAttr' on <module 'torch_geometric.data.data' from '/opt/conda/lib/python3.7/site-packages/torch_geometric/data/data.py'>
If I need reinstall this pyG?

**Comment by EdisonLeeeee** [comment, marked as answer]
Could you remove the processed dir and try it again? That seems the reddit dataset was processed by an earlier version of PyG.

**Reply by smujiang** [reply]
Thank you. Regenerate the dataset solved my problem.

**Reply by jeongwhanchoi** [reply]
Thank you!
